### ISSUE TITLE
QtGui ImportError, ModuleNotFoundError (3.0.5)

### ISSUE THREAD

**Issue by gb4944** [psychopy/psychopy#2297]
Version: latest standalone version (3.0.5) on Win64.

When trying to run any experiment, I get the following error:
```
File "C:\Program Files (x86)\PsychoPy3\lib\site-packages\psychopy\gui\qtgui.py", line 15, in <module>
	from PyQt4 import QtGui  
ModuleNotFoundError: No module named 'PyQt4'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "H:\cdots\cdots_lastrun.py", line 18, in <module>
	from psychopy import locale_setup, sound, gui, visual, core, data, event, logging, clock
  File "C:\Program Files (x86)\PsychoPy3\lib\site-packages\psychopy\gui\__init__.py", line 33, in <module>
	from .qtgui import *
  File "C:\Program Files (x86)\PsychoPy3\lib\site-packages\psychopy\gui\qtgui.py", line 19, in <module>
	from PyQt5 import QtWidgets
ImportError: DLL load failed: The specified procedure could not be found.
```

The problem disappears when switching back to 3.0.2.

**Comment by JulianKLy** [comment]
I encountered the same problem: 
```Traceback (most recent call last):
File "C:\Users...\experiment2_b_lastrun.py", line 14, in <module>
	from psychopy import locale_setup, sound, gui, visual, core, data, event, logging, clock
File "C\Program Files (x86)\PsychoPy\lib\site-packages\psychopy\gui\_init_.py", line 33, in <module>
	from .qtgui import *
Fine "C\Program Files...\gui\qtgui.py", line 19, in <module>
	from PyQt5 import QtWidgets
ImportError: DLL load failed: The specific procedure could not be found```

**Comment by peircej** [comment]
That's weird. I'll look into why that one has (suddenly) failed to package. Thanks for the heads-up that it isn't working

**Comment by peircej** [comment]
It looks like this is caused by a missing value in the windows PATH variable. Try editing you path variable (system settings) and add `<psychopy location>/DLLs`

**Comment by peircej** [comment]
(sorry the close of the issue was an accident - let me know if this is sufficient to fix)

**Comment by cathelaing** [comment]
Hello,

I'm also having this problem. Would you mind clarifying where I should edit the path variable? (ie. I'm not sure where I can edit system settings...)

Many thanks!

Catherine

**Comment by m-topor** [comment]
Hi! 
I am having the exact same problem and identical errors as gb4944. 
I looked at the path variables and the correct one i.e. <psychopy location>/DLLs was already added. 

Thanks!

**Comment by felipeviegas** [comment]
Same error to me:

Traceback (most recent call last):
  File "C:\Program Files (x86)\PsychoPy3\lib\site-packages\psychopy\gui\qtgui.py", line 15, in <module>
    from PyQt4 import QtGui  
ModuleNotFoundError: No module named 'PyQt4'

During handling of the above exception, another exception occurred:
´´´
Traceback (most recent call last):
  File "C:\Users\fisiologia_2\Documents\teste.py", line 14, in <module>
    from psychopy import locale_setup, sound, gui, visual, core, data, event, logging, clock
  File "C:\Program Files (x86)\PsychoPy3\lib\site-packages\psychopy\gui\__init__.py", line 33, in <module>
    from .qtgui import *
  File "C:\Program Files (x86)\PsychoPy3\lib\site-packages\psychopy\gui\qtgui.py", line 19, in <module>
    from PyQt5 import QtWidgets
ImportError: DLL load failed: Não foi possível encontrar o procedimento especificado.
´´´
Even with the most simple experiment. It begans with trying to open an experiment created on a Mac, not sure about the Psychopy version, on a newly installed version of Psychopy 3.05 on a Windows. After not being successful, I tried to show one letter at the screen. Same error. 

Should I try version 3.02? I am very discouraged to update my mac version of Psychopy, although I would like to try Pavlovia...

**Comment by mariafranciscva496** [comment]
> (sorry the close of the issue was an accident - let me know if this is sufficient to fix)

Hello I have the exact same error, and I already tried to put the C:\Program Files (x86)\PsychoPy3\DLLs into my path but it didn't work, do you have any more ideas? thank you

**Comment by felipeviegas** [comment]
> > (sorry the close of the issue was an accident - let me know if this is sufficient to fix)
> 
> Hello I have the exact same error, and I already tried to put the C:\Program Files (x86)\PsychoPy3\DLLs into my path but it didn't work, do you have any more ideas? thank you

It didn't work for me either...

**Comment by gb4944** [comment]
> It looks like this is caused by a missing value in the windows PATH variable. Try editing you path variable (system settings) and add `<psychopy location>/DLLs`

I have the same issue as others above, only going back to an older version has worked so far.

**Comment by mariafranciscva496** [comment]
> > It looks like this is caused by a missing value in the windows PATH variable. Try editing you path variable (system settings) and add `<psychopy location>/DLLs`
> 
> I have the same issue as others above, only going back to an older version has worked so far.

Hello, thank you for your response. An older version of what? Pyqt?

**Comment by gb4944** [comment]
> > > It looks like this is caused by a missing value in the windows PATH variable. Try editing you path variable (system settings) and add `<psychopy location>/DLLs`
> > 
> > 
> > I have the same issue as others above, only going back to an older version has worked so far.
> 
> Hello, thank you for your response. An older version of what? Pyqt?

Apologies, I should have added that. I meant an older version of PsychoPy; I went back to 3.0.2, since that was the last version I successfully tested the experiment with prior to updating to 3.0.5.

**Comment by mariafranciscva496** [comment]
> > > > It looks like this is caused by a missing value in the windows PATH variable. Try editing you path variable (system settings) and add `<psychopy location>/DLLs`
> > > 
> > > 
> > > I have the same issue as others above, only going back to an older version has worked so far.
> > 
> > 
> > Hello, thank you for your response. An older version of what? Pyqt?
> 
> Apologies, I should have added that. I meant an older version of PsychoPy; I went back to 3.0.2, since that was the last version I successfully tested the experiment with prior to updating to 3.0.5.

Thank you, but I heard older versions of PsychoPy have problems when it comes to have auditory and visual stimuli simultaneosly, do you know if that version has?

**Comment by peircej** [comment]
Well, I'll have another look at this tomorrow to try and work out what might be going on (and what might have changed between 3.0.2 and 3.0.5.
@felipeviegas the good thing about the mac versions is that you _can_ happily install multiple versions together because the app bundles don't interact. With windows the previous version gets uninstalled with the install of a new version, but that doesn't happen on mac

**Comment by felipeviegas** [comment]
> Well, I'll have another look at this tomorrow to try and work out what might be going on (and what might have changed between 3.0.2 and 3.0.5.
> @felipeviegas the good thing about the mac versions is that you _can_ happily install multiple versions together because the app bundles don't interact. With windows the previous version gets uninstalled with the install of a new version, but that doesn't happen on mac

Thank you so much @peircej . I'll try version 3.

**Comment by fidadoma** [comment]
Hi, I've run into the same issue. I found out that the error was present only in 64bit version, not 32 bit. hope it helps in finding the error

**Comment by JulianKLy** [comment]
I re-installed 3.0.4 (Windows) and it worked in that version too

**Comment by felipeviegas** [comment]
It worked for me on 3.0.2, as suggested. First time Python was crashing, though. But after reinstall (3.0.2), everything was ok. On Mac, no problems with 3.0.5, everything works just fine.

**Comment by peircej** [comment]
Many thanks @fidadoma knowledge that this was a 64/32bit issue really does help me get to the bottom of it! It turns out there was a problem in one of the PyQt releases for 64bit wheels and I've now been able to remove that and install a new one. 

This will be fixed in the next release of PsychoPy standalone

**Comment by peteruhrig** [comment]
I can confirm the error is gone for me in the Windows standalone 64bit version 3.0.6. Thanks a lot!

**Comment by hoechenberger** [comment]
Fixed in 3.0.6, closing.
